app/models/product_import/entry_processor.rb

```diff
@@ -46,7 +46,10 @@ def count_existing_items
 
         products_count =
           if settings.importing_into_inventory?
-            VariantOverride.where('variant_overrides.hub_id IN (?)', enterprise_id).count
+            VariantOverride.where(
```

Comment:
I thought `VariantOverride.where(hub_id: [enterprise_id])` would work as well. I just checked the class and this implementation is used for a scope already. You can write `VariantOverride.for_hubs([enterprise_id])`.